src/frontend/styles/components/writing.pcss

```diff
@@ -17,16 +16,40 @@
 
   &__left {
     margin: auto 0;
+    display: flex;
     color: var(--color-text-second);
+    gap: 10px;
 
-    & span {
-      margin-right: 10px;
+    @media(--mobile) {
+      flex-flow: column;
+    }
+
+    & > * {
+      flex: 0 1 33.3%;
+    }
+
+    label {
+      display: block;
+      margin-bottom: 6px;
     }
-  }
 
-  select {
-    max-width: 100px;
-    vertical-align: bottom;
+    select, input {
+      position: relative;
+      padding: 10px;
+      width: 100%;
+      border: none;
+      border-radius: 8px;
+      background-color: #F3F6F8;
+      box-sizing: border-box;
+      appearance: none;
+    }
+
+    select {
+      padding-right: 36px;
```

Comment:
padding right seems too big.

Stage:
<img width="776" alt="image" src="https://user-images.githubusercontent.com/3684889/174046109-35073062-6102-45c8-b5ff-19c5a6f61e97.png">

Figma: 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/3684889/174046177-e0cc05a1-998a-4a15-b262-bc3c5223d4a5.png">

Reply:
The caret icon was bigger than Figma so downsized it